pom.xml

```diff
@@ -33,26 +33,27 @@
         <jboss.home>IF-NOT-DEFINED-WILDFLY-WILL-BE-DOWNLOADED-UNZIPPED-AND-USED-AUTOMATICALLY</jboss.home>
         <maven.test.redirectTestOutputToFile>false</maven.test.redirectTestOutputToFile>
         <version.io.rest-assured>5.0.1</version.io.rest-assured>
-        <version.javax.servlet.javax.servlet-api>3.0.1</version.javax.servlet.javax.servlet-api>
+        <version.jakarta.servlet.jakarta.servlet-api>5.0.0</version.jakarta.servlet.jakarta.servlet-api>
         <version.junit>4.13.1</version.junit>
         <version.org.apache.maven.plugins.maven-release-plugin>3.0.0-M1</version.org.apache.maven.plugins.maven-release-plugin>
-        <version.org.eclipse.microprofile>4.0.1</version.org.eclipse.microprofile>
-        <version.org.jboss.arquillian>1.5.0.Final</version.org.jboss.arquillian>
-        <version.org.jboss.wildfly.dist>20.0.0.Final</version.org.jboss.wildfly.dist>
-        <version.org.wildfly.arquillian>3.0.1.Final</version.org.wildfly.arquillian>
+        <version.org.eclipse.microprofile>5.0</version.org.eclipse.microprofile>
+        <version.org.jboss.arquillian>1.7.0.Alpha12</version.org.jboss.arquillian>
+        <version.org.jboss.wildfly.dist>27.0.0.Alpha4</version.org.jboss.wildfly.dist>
+        <version.org.wildfly.arquillian>5.0.0.Alpha5</version.org.wildfly.arquillian>
         <version.org.fusesource.jansi>1.18</version.org.fusesource.jansi>
         <!-- wildfly-core update needed to have the launcher version with BootableJarCommandBuilder used to start
         the server up -->
-        <version.org.wildfly.core>19.0.0.Beta8</version.org.wildfly.core>
+        <version.org.wildfly.core>19.0.0.Beta15</version.org.wildfly.core>
         <version.org.wildfly.extras.creaper>1.6.1</version.org.wildfly.extras.creaper>
         <version.org.yaml.snakeyaml>1.26</version.org.yaml.snakeyaml>
         <version.com.google.code.gson>2.8.6</version.com.google.code.gson>
         <version.com.fasterxml.jackson.core.jackson-databind>2.9.9.1</version.com.fasterxml.jackson.core.jackson-databind>
-        <version.org.glassfish.javax.json>1.1</version.org.glassfish.javax.json>
+        <version.org.glassfish.javax.json>1.1.4</version.org.glassfish.javax.json>
```

Comment:
Can we remove javax.json at all? Or is it still used somewhere?

Reply:
sure, tnks for spotting this